libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -190,38 +190,38 @@ const AP_Param::GroupInfo AP_SerialManager::var_info[] = {
     AP_GROUPINFO("1_PROTOCOL",  1, AP_SerialManager, state[1].protocol, DEFAULT_SERIAL1_PROTOCOL),
 
     // @Param: 1_BAUD
-    // @DisplayName: Telem1 Baud Rate
+    // @DisplayName: Serial1 Baud Rate
```

Comment:
PIXHAWK and CUBE systems have TELEM1 and TELEM2 labeled on their cases.
I think it would be better to match the display accordingly.

Reply:
marking as resolved because change removed